pep-0387.txt

```diff
@@ -3,8 +3,8 @@ Title: Backwards Compatibility Policy
 Version: $Revision$
 Last-Modified: $Date$
 Author: Benjamin Peterson <benjamin@python.org>
-BDFL-Delegate: Brett Cannon (on behalf of the steering council)
```

Comment:
Since this PEP is still "Active" it seems reasonable to change it

Reply:
Yeah, I didn't mass-change it; in this case though this PEP is explicitly stated to be approved `on behalf of the Steering Council` (rather than the BDFL) so I wanted to ensure that the header reflected that.